src/ServiceControl.Monitoring/WebApplicationBuilderExtensions.cs

```diff
@@ -115,7 +111,13 @@ internal static void ConfigureEndpoint(EndpointConfiguration config, Func<ICriti
 
         config.UseSerialization<NewtonsoftJsonSerializer>();
         config.UsePersistence<NonDurablePersistence>();
+
+        var recoverability = config.Recoverability();
+        recoverability.Immediate(c => c.NumberOfRetries(3));
```

Comment:
Do we need to specify the immediate retries and why not simply disable the delayed ones that we don't need? Looking at the change we now effectively went from 5 (default) immediate to 3 and I'm wondering what the implications could be.

Reply:
Ah you basically aligned it with SC instances I think that is fine.